access.go

```diff
@@ -345,27 +338,27 @@ func (s *Server) handleRefreshTokenRequest(w *Response, r *http.Request) *Access
 	var err error
 	ret.AccessData, err = w.Storage.LoadRefresh(ret.Code)
 	if err != nil {
-		w.SetError(E_INVALID_GRANT, "")
+		w.SetError(E_SERVER_ERROR, "")
 		w.InternalError = err
 		return nil
 	}
 	if ret.AccessData == nil {
-		w.SetError(E_UNAUTHORIZED_CLIENT, "")
+		w.SetError(E_INVALID_GRANT, "")
 		return nil
 	}
 	if ret.AccessData.Client == nil {
-		w.SetError(E_UNAUTHORIZED_CLIENT, "")
+		w.SetError(E_INVALID_GRANT, "")
 		return nil
 	}
 	if ret.AccessData.Client.GetRedirectURI() == "" {
-		w.SetError(E_UNAUTHORIZED_CLIENT, "")
+		w.SetError(E_INVALID_GRANT, "")
 		return nil
 	}
 
 	// client must be the same as the previous token
 	if !CheckClientID(ret.AccessData.Client, ret.Client.GetID()) {
-		w.SetError(E_INVALID_CLIENT, "")
-		w.InternalError = errors.New("Client id must be the same from previous token")
+		w.SetError(E_INVALID_GRANT, "")
+		w.InternalError = errors.New("client ID must be the same from previous token")
```

Comment:
I would prefer to have the `ID` lowercase so it is consistent with others.

Reply:
sure